modules/repository/src/test/java/com/intuit/wasabi/repository/impl/database/DatabaseAnalyticsTest.java

```diff
@@ -51,13 +51,13 @@
     public void setup() throws SQLException {
         when(transactionFactory.newTransaction()).thenReturn(transaction);
         when(transactionFactory.getDataSource()).thenReturn(dataSource);
-        databaseAnalytics = spy(new DatabaseAnalytics(transactionFactory, flyway));
+        databaseAnalytics = spy(new DatabaseAnalytics(transactionFactory, flyway, "com/intuit/wasabi/repository/impl/mysql/migration"));
```

Comment:
nit: for unit tests, i like using fake/mock data where possible

not sure if this is a pure unit test? or an integration test pretending to be a unit test?

not a blocker either way